doc/project/release_process.rst

```diff
@@ -85,24 +86,154 @@ At that point, the whole process starts over again.
+|          | inconsistent results.                                            |
++----------+------------------------------------------------------------------+
+| Minor    | No crashes or damage to usability of the system and desired      |
+|          | results can be easily obtained by working around the defects.    |
++----------+------------------------------------------------------------------+
+| Cosmetic | Related to look and feel of the system.                          |
++----------+------------------------------------------------------------------+
+
+Severity shall be set based on the context of the bug and not in relation to the
+overall impact on the codebase. For example, a bug in an optional subsystem that
+results in a crash shall be marked as `critical` even if the bug does not impact
+every user of Zephyr.
+
+Blocker Bugs
+============
+
+Blocker bug process kicks in during the release process and is in effect after the
+feature freeze milestone. An issue labeled as a blocker practically blocks a
+release from happening. All blocker bugs shall be resolved before a release is
+created.
 
-The current backlog of prioritized bugs shall be used as a quality metric to
-gate the final release. The following counts shall be used:
+A bug that is granted `blocker` status can be merged to 'main' and included in
+the release all the way until the hard freeze (RC3) milestone.
 
-.. csv-table:: Bug Count Release Thresholds
-   :header: "High", "Medium", "Low"
-   :widths: auto
+Bugs of moderate severity and higher are typically the candidates to be promoted
+to blocker bugs.
 
+The following guidelines shall be followed for release blocker bugs:
 
-   "0","<20","<50"
+- Only mark bugs as blockers if the software (Zephyr) must not be released with
+  the bug present.
+- All collaborators can add or remove blocking labels.
+- Evaluate bugs as potential blockers based on their severity and prevalence.
+- Provide detailed rationale whenever adding or removing a blocking label.
+- Ensure all blockers have the milestone tagged.
+- Release owners have final say on blocking status; contact them with any questions.
 
-.. note::
+Hardware Support Tiers
+======================
+
+Tier 0: Emulation Platforms
+---------------------------
+
+- Failures can block Pull Requests.
+- Supported by the Zephyr project itself, commitment to fix bugs in releases.
+- One is required for each new architecture.
+- Bugs reported against platforms of this tier are to be evaluated and treated as
+  a general bug in Zephyr and should be dealt with the highest priority.
+
+Tier 1: Supported Platforms
+---------------------------
+
+- Commitment from a specific team to run tests using twister device
+  testing for the "Zephyr compatibility test suite" (details TBD)
+  on a regular basis using open drivers.
+- Commitment to fix bugs in time for releases. Not supported by "Zephyr Project"
+  itself.
```

Comment:
Should we give an indication of expected time-to-resolution? Just to give a guideline of how a support team should be staffed. Commitment here means from the platform vendor, right?

Reply:
> Commitment here means from the platform vendor, right?

yes.